src/Generator/GeneratorFactory.php

```diff
@@ -12,40 +12,54 @@ class GeneratorFactory
 {
     /**
      * @param Config::BACKEND_* $backend
-     * @param Config::OPERATION_* $operation
+     * @param Config::OPERATION_ACTION_* $operation
+     * @param Config::OPERATION_TYPE_* $operationType
      * @param Config::SOURCE_* $source
+     * @param Config::FILE_STORAGE_*|null $fileStorage
      * @throws UserException
      */
     public function factory(
         string $backend,
         string $operation,
-        string $source
+        string $operationType,
+        string $source,
+        ?string $fileStorage
     ): GeneratorInterface {
         if ($backend === Config::BACKEND_SNOWFLAKE) {
-            if ($operation === Config::OPERATION_IMPORT_FULL) {
-                if ($source === Config::SOURCE_FILE_ABS) {
-                    return new Generator\Snowflake\ImportFull\FromAbsGenerator();
+            if ($operation === Config::OPERATION_ACTION_IMPORT) {
```

Comment:
tady jsi to možná mohl udělat přes && ať to není milion zanoření, ale asi OK :)

Reply:
nelibila se mi ani jedna varianta 🤷 tohle je aspon hezky rozsiritelny...

Reply:
👍